certbot/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 Certbot adheres to [Semantic Versioning](https://semver.org/).
 
+## 1.24.1 - master
+
+### Fixed
+
+* The packages used in our snaps were updated in response to
+  https://ubuntu.com/security/notices/USN-5320-1.
+
+This release was only done for our snaps, however, all of our distribution
+mechanisms were updated.
```

Comment:
I have trouble understanding what the second half of this sentence means. Like, do non-snap users need to care?

Reply:
This release wouldn't be important to non-snap users, however, I was planning on pushing an update everywhere for simplicity. If you look at https://github.com/certbot/certbot/pull/9234/commits/b5503e91d19c4f740077641f8a2bc68bae32b33b or my crossed out text in Mattermost, I was initially planning on limiting the release to snap & docker users, but I'm hesitant to change our release process and do release code in a way we've never done before right before I disappear for a week and a half. We could revisit that plan or we could try to clarify this text to flag that this release is really only useful to our snap users. What do you think?

Reply:
Ah, I misread it as "we're only releasing the snap", rather than "this release was only done [for the sake of] our snaps".

I think clarifying the text would be useful. 👍

Reply:
Ah OK. Sounds good.

Feel free to offer an alternative if you still don't like the new text.